virtual_rainforest/entry_points.py

```diff
@@ -0,0 +1,67 @@
+"""The `virtual_rainforest.entry_points` module.
+
+This module defines the entry points to the virtual_rainforest package. At the moment a
+single entry point is defined `vr_run`, which simply configures and runs a virtual
+rainforest simulation based on a set of configuration files.
+"""
+
+import argparse
+import textwrap
+
+import virtual_rainforest as vr
+from virtual_rainforest.core.config import ConfigurationError
+from virtual_rainforest.core.logger import log_and_raise
+from virtual_rainforest.main import vr_run
+
+
+def _vr_run_cli() -> None:
+    """Configure and run a Virtual Rainforest simulation.
+
+    This program sets up and runs a simulation of the Virtual Rainforest model. At
+    present this is incomplete. At the moment a set of configuration files are read in
+    based on user supplied paths, these are converted to a single configuration file,
+    which is then output for further reference. This combined configuration is then used
+    to initialise a set of models.
+    """
+
+    # Check function docstring exists, as -OO flag strips docstrings I believe
+    desc = textwrap.dedent(_vr_run_cli.__doc__ or "Python in -OO mode: no docs")
+    fmt = argparse.RawDescriptionHelpFormatter
+    parser = argparse.ArgumentParser(description=desc, formatter_class=fmt)
+
+    parser.add_argument(
+        "cfg_paths", help="Paths to config files and/or folders.", nargs="*"
+    )
+    parser.add_argument(
+        "-o",
+        "--output_folder",
+        default=".",
+        type=str,
+        help="Folder that the output config file should be saved in.",
+        dest="output_folder",
+    )
+    parser.add_argument(
+        "-n",
```

Comment:
Yeah I agree, that is a bit of a hangover from when we were reading files from a specified config folder. So this can probably all be collapsed into a single merge option